README.md

```diff
@@ -35,6 +35,12 @@ From the root of this repository, run the following commands:
 kind create cluster --name konflux --config kind-config.yaml
 ```
 
+**Note:** Kind uses docker by default, so if you want to use podman, run the command the following way:
+
+```bash
+KIND_EXPERIMENTAL_PROVIDER=podman kind create cluster --name konflux --config kind-config.yaml
```

Comment:
At least when only podman is installed, this is not required. I.e. it uses podman automatically.

Reply:
we can somehow reflect the fact that if both docker and podman are installed, kind will use docker as a default, and if we want it to run with podman, thats the command to run

Reply:
Hmmm. Interesting! I had the issue because I have both installed. I can drop it or rephrase the note. Whatever you two prefer!

Reply:
What I try to keep in mind is that the flow is already not short. I'd prefer we don't add things which are not required.

In this case, if it works, I don't mind if the users use podman or docker. We support both, so there's no incentive to make them lean toward podman.

Reply:
> Hmmm. Interesting! I had the issue because I have both installed. I can drop it or rephrase the note. Whatever you two prefer!

Did you see issues when using docker?

Reply:
I didn't use docker. I have it installed but not enabled. That's why I got the error and I looked for how to use podman with kind instead of docker

Reply:
Ok!

Reply:
Ok!